### ISSUE TITLE
angularx-social-login doesn't show photourl with facebook

### ISSUE THREAD

**Issue by striderpapo** [abacritt/angularx-social-login#361]
hello everyone, i'm new here and i'm trying to use angularx-social-login, the problem is it doesn't show facebook image and i followed all the steps,Thank you all for your attention
![facenook](https://user-images.githubusercontent.com/78055747/105921789-f0ba3880-5ffe-11eb-9aca-3504c8fbe942.png)

**Comment by jaibatrik** [comment]
I think there were some changes to Facebook policy regarding photo URLs. Probably the app needs additional permission or something. Could you please search on this?

**Comment by Aerofusical** [comment]
That is correct, Facebook changed the way the user photo is obtained and it also has a TTL.

The following can be used to fetch the correct Facebook user image:
```
<img src="{{ user.photoUrl }}&access_token={{user.authToken}}">
```

**Comment by Selindek** [comment]
> That is correct, Facebook changed the way the user photo is obtained and it also has a TTL.
> 
> The following can be used to fetch the correct Facebook user image:
> 
> ```
> <img src="{{ user.photoUrl }}&access_token={{user.authToken}}">
> ```

That sniplet should be added to the FacebookLoginProvider. Otherwise - if I allow login with different providers - I not onlu needed to add that extra parameter to the imgUrl, but also need to check what is the current user's providerId and make the modification ONLY if it's FACEBOOK.

**Comment by jaibatrik** [comment]
Merged the PR and released to npm - angularx-social-login@3.5.7.
